docs/journalist/faq.rst

```diff
@@ -58,30 +58,64 @@ For more about the security features of Qubes, see
+
+Because the air-gapped SVS has no Internet access, updates can only be performed using
+another computer and a USB drive. In practice, newsrooms may not update their SVS
+in a timely manner, which can significantly worsen its security posture.
+
+In SecureDrop Workstation, any document received via SecureDrop is opened in a
+disposable VM that has no Internet access and no access to other files submitted
+via SecureDrop. The encryption keys are stored in a separate, networkless VM
+from the SecureDrop Client app.
+
+Because SecureDrop Workstation has Internet access, updates can be applied
+automatically as soon as they are available. SecureDrop Workstation enforces this
+by downloading and applying updates before the user logs into SecureDrop.
+
+SecureDrop Workstation uses hardware-assisted virtualization, which allows us
+to use custom kernels for its VMs. These custom kernels use the
+`grsecurity <https://grsecurity.net/>`__ patches which are also used on the
+SecureDrop servers, and provide additional mitigation against security
+vulnerabilities.
+
+An attacker able to exploit vulnerabilities in Qubes OS or Xen-based bare metal
+virtualization (likely in combination with other vulnerabilities, e.g., in a
+viewer application) may be able to exfiltrate information directly to the Internet.
+Qubes closely `tracks <https://www.qubes-os.org/security/xsa/>`__ any security
+vulnerabilities that may impact it, and the automatic update mechanism helps to
+ensure that, in the event of a vulnerability, every SecureDrop Workstation can be
+patched as quickly as possible.
 
 Can I install custom software on SecureDrop Workstation?
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 Right now, the pilot project is designed to make the journalist experience
 easier by combining the functionality of the Journalist Workstation and Secure
 Viewing Station. The main focus is making sure that checking SecureDrop is
-easier and faster. While we hope to add advanced tooling and document-
-processing options down the line, at this time we request that you leave the
-workstation configured the way it was provisioned, and hold off on installing
-additional software or changing the functionality or configuration of any of
-the VMs. If you have specific needs that you'd like to discuss with us, open an
-issue `in our support portal`_ or send us a `GPG-encrypted email`_ at
-support@freedom.press.
+easier and faster.
+
+While we hope to add advanced tooling and document-processing options down the line,
+at this time we request that you leave the workstation configured the way it was
+provisioned, and hold off on installing additional software or changing the
+functionality or configuration of any of the VMs. If you have specific needs
+that you would like to discuss with us, pelase open an issue `in our support portal`_
```

Comment:
*please*

Reply:
haha, good catch

Reply:
Request using a simpler word than "provision" in all journalist-facing content.

Reply:
in this instance, changed to "change the configuration of the workstation or install additional software on it"